spec/features/search/dish_search_spec.rb

```diff
@@ -0,0 +1,29 @@
+feature "Dish search" do 
+
+  before(:each) do 
+    search_reindex "Dish"
```

Comment:
Looks like `"Dish"` doesn't need to be a string.

`search_reindex Dish` ought to work.